### DISCUSSION TITLE
After App Update no Dark Mode

### DISCUSSION THREAD

**Discussion by jb-alvarado** [tauri-apps/tauri#6147] [Q&A]
Hello,
I uses the default color schema from the starter Project under fedora 37 (Gnome). My desktop environment is set to dark mode and when I open the tauri app, it is also dark.

But I use also the update function from tauri and when a new update is available and I install it, it comes the popup to restart the app, and after clicking yes, the app starts in light mode.

Is there any trick, to prevent this?

**Comment by FabianLars** [comment]
This sounds like something that should be a bug report instead 🤔 Could you convert this discussion to an issue instead please?

**Reply by jb-alvarado** [reply]
Ok, is done in #6149
